diff --git a/supported_wheels.py b/supported_wheels.py
index 41abc37..454fe9c 100755
--- a/supported_wheels.py
+++ b/supported_wheels.py
@@ -28,7 +28,10 @@ def tags_for(fname):
 
 
 def main():
-    supported = set(get_supported())
+    supported = {
+         (tag.interpreter, tag.abi, tag.platform) if not isinstance(tag, tuple) else tag
+         for tag in get_supported()
+    }
     for fname in sys.argv[1:]:
         tags = set(tags_for(fname))
         if supported.intersection(tags):
